src/components/UI/Slider/SliderWithMarkers.vue

```diff
@@ -40,16 +40,9 @@ export default {
       clickAfterDragEnded: false, // indicates whether a marker click was invoked right after it was dragged
     };
   },
-  watch: {
-    // watch whenever "isDragDisabled" changes and update the marker
-    // colors accordingly
-    isDragDisabled() {
-      this.setDisabledMarkerColor();
-    },
-  },
   created() {
     // set marker colors when instantiated
-    this.setDisabledMarkerColor();
+    // this.setDisabledMarkerColor();
```

Comment:
can remove this and update with a new comment saying what to do when the component is created